zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -63,6 +68,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.zookeeper.KeeperException.Code.CONNECTIONLOSS;
+import static org.apache.zookeeper.KeeperException.Code.OK;
+import static org.apache.zookeeper.KeeperException.Code.SESSIONMOVED;
```

Comment:
Don't forget to remove these unused static imports.

FYI, actually I would prefer not to import the static constant. You could just use `KeeperException.Code.CONNECTIONLOSS` for better readability. Ref: https://docs.oracle.com/javase/7/docs/technotes/guides/language/static-import.html

Reply:
done.